prow/scripts/kyma-testing.sh

```diff
@@ -71,6 +71,23 @@ cts::delete() {
 
 }
 
+inject_addons_if_necessary() {
+  tdWithaddon=$(${kc} get td -l testing.kyma-project.io/require-testing-addon=true -o custom-columns=NAME:.metadata.name --no-headers=true)
```

Comment:
please provide `--all-namespaces` flag

Reply:
`tdWithaddon` -> `tdWithAddon`

Reply:
good catch! without that tomorrow I will be mentioned as a person who broke our pipeline :D